config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -0,0 +1,44 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cluster-bundle:
+  - name: gcp-k8s-cluster-bundle-presubmit
+    always_run: true         # Run for every PR, or only when requested.
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180918-762ed5b64-master
+        imagePullPolicy: Always
+        command:
+        - "bazel"
+        - "test"
+        - "..."
```

Comment:
You may need to move the arguments into the `args` field. I'm not sure how this will behave.

Reply:
Okay.

Reply:
Official docs: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes

Though with the podutils we hijack the `commmand` field such that it looks something like:
`[podutils-entrypoint-binary, command ...]` `[args... ]`